meta-iotqa/lib/oeqa/runtime/peripherals/mraa/mraa_gpio.py

```diff
@@ -18,30 +34,75 @@ def setUp(self):
         else:
             raise unittest.SkipTest(output)
 
-    def test_gpio(self):
+    def check_gpio_value(self):
+        cmd = "cat /sys/class/gpio/gpio20/value".split()
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        return int(output)
+
+    def mraa_gpio_get_value(self):
+        status, output = self.target.run("mraa-gpio get " + self.pin)
+        output = str(output)
+        string1, string2 = output.split("=")
+        string2 = int(re.search(r'\d+', string2).group())
```

Comment:
why is this needed? I believe you should also check ```status``` is OK before returning anything.